### PR TITLE
refactor: update to latest near-sdk & serde-json

### PR DESCRIPTION
**Pull request by chadoh**
Fixes near/devx#145

Some of the changes here came from running the Rust autoformatter that is [built into my editor](https://github.com/rust-lang/rust.vim#formatting-with-rustfmt)

### CODE REVIEW

**Comment by chadoh** [comment]
@evgenykuzyakov let me know if I should add `#[payable]` to match [`near-sdk-rs/examples` source](https://github.com/near/near-sdk-rs/blob/8830b97aed89cf982e8aaffdded6a4dfee6482a7/examples/status-message/src/lib.rs#L17)

**Comment by evgenykuzyakov** [comment]
No you don't need `#[payable]`. It was added to make sure it compiles within SDK.

File: Cargo.toml [hunk 1]
```diff
@@ -9,8 +9,8 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
-near-bindgen = { version = "0.6.0" }
+serde_json = "1.0.52"
+near-sdk = "0.9.2"
 borsh = "0.6.1"
 wee_alloc = "0.4.5"
```
Comment (evgenykuzyakov) [inline Cargo.toml:15]:
```suggestion
serde = { version = "*", features = ["derive"] }
serde_json = "*"
borsh = "*"
near-sdk = "0.9.2"
wee_alloc = { version = "0.4.5", default-features = false, features = [] }
```
This way you can make sure there are no different versions of common libraries.

File: src/lib.rs [hunk 2]
```diff
@@ -1,11 +1,11 @@
 use borsh::{BorshDeserialize, BorshSerialize};
-use near_bindgen::{env, metadata, near_bindgen};
+use near_sdk::{env, metadata, near_bindgen};
 use std::collections::HashMap;
 
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-metadata!{
+metadata! {
```
Comment (evgenykuzyakov) [inline src/lib.rs:8]: You also don't need metadata. Please remove it

File: Cargo.toml (hunk 1 above)
Comment (chadoh) [reply Cargo.toml]: Thanks!

File: src/lib.rs (hunk 2 above)
Comment (chadoh) [reply src/lib.rs]: I removed the `metadata` wrapper but kept the content within – is that what you meant? The tests did not pass if I removed the whole block.

Comment (evgenykuzyakov) [reply src/lib.rs]: Yes. Just remove the macro

**Pull request merged**